internal/pkg/service/buffer/store/model/mapper/mapper_test.go

```diff
@@ -0,0 +1,207 @@
+package mapper
+
+import (
+	"testing"
+
+	"github.com/keboola/go-utils/pkg/wildcards"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/encoding/json"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/api/gen/buffer"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/store/key"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/store/model"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/store/model/column"
+)
+
+func TestMapperReceiverPayloadToModel(t *testing.T) {
```

Comment:
Mohol by si to nazvat nejak rovnako... test by sa mal volat podla funkcie, neviem ci ti to aj IDE ukaze alebo nie.

Tu mas `ReceiverPayloadToModel`, ale funkcia pouzita nizsie je `ReceiverModelFromPayload`.